pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -94,19 +94,35 @@ in mkDerivation {
         mkdir -p $out/share/icons/hicolor/$path/apps
         cp $icon $out/share/icons/hicolor/$path/apps/us.zoom.Zoom.png
     done
-
-    ln -s $out/share/zoom-us/zoom $out/bin/zoom-us
   '';
 
+  # $out/share/zoom-us isn't in auto-wrap directories list, need manual wrapping
+  dontWrapQtApps = true;
+
   qtWrapperArgs = [
     ''--prefix PATH : ${makeBinPath [ coreutils glib.dev pciutils procps qttools.dev utillinux ]}''
     ''--prefix LD_PRELOAD : ${libv4l}/lib/libv4l/v4l2convert.so''
+    # --run "cd ${builtins.placeholder "out"}/share/zoom-us"
+    # ^^ unfortunately, breaks run arg into multiple array elements, due to
+    # some bad array propagation. We'll do that in bash below
   ];
 
+  postFixup = ''
+    # Zoom expects "zopen" executable (needed for web login) to be present in CWD. Or does it expect
+    # everybody runs Zoom only after cd to Zoom package directory? Anyway, :facepalm:
+    qtWrapperArgs+=( --run "cd ${builtins.placeholder "out"}/share/zoom-us" )
+
+    for app in ZoomLauncher zopen zoom; do
```

Comment:
:facepalm also for zopen
```
❯ ./result/share/zoom-us/zopen --help
xdg-open - opens a file or URL in the user's preferred application
```

it's literally xdg-open.

Reply:
It uses cross-platform https://doc.qt.io/qt-5/qdesktopservices.html#openUrl, which uses `xdg-open` under the hood (confirmed with strace)